#! /usr/bin/env python

PACKAGE = 'point_cloud_filter'

import os
import sys
sys.path.insert(0, os.path.dirname(__file__))
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator ()

gen.add("num_threads", int_t, 0, "Number of threads for normal computation", 4)
gen.add("normal_search_method", str_t, 0, "Search method: search_radius, search_knn", "search_knn")
gen.add("normal_search_radius", double_t, 0, "Search radius for normal computation", 0.3)
gen.add("normal_search_knn", int_t, 0, "Search nearest neighbours for normal computation", 20)

exit (gen.generate (PACKAGE, "pcl_ros", "NormalComputation"))
